core/string/locales.h

```diff
@@ -1111,7 +1111,7 @@ static const char *script_list[][2] = {
 	{ "Meitei Mayek", "Mtei" },
 	{ "Multani", "Mult" },
 	{ "Myanmar / Burmese", "Mymr" },
-	{ "​Nag Mundari", "Nagm" },
+	{ "Nag Mundari", "Nagm" },
```

Comment:
What was the issue here? Neither GitHub nor the .diff view make it obvious. Was that an invisible control character?

Reply:
There's copy-pasted `zero width space` character before the `Nag` (probably was used as part of table formatting on the Unicode release page or wiki).

Reply:
Might be worth doing a pass at grepping the whole codebase for this kind of non-visible Unicode characters, to see if we spot more (and maybe even add to pre-commit checks eventually to prevent introducing new ones).